src/lib/Components/Inbox/Conversations/Preview/ArtworkPreview.tsx

```diff
@@ -86,10 +77,11 @@ export class ArtworkPreview extends React.Component<Props, any> {
               {artwork.artist_names}
             </SerifText>
             <TitleAndDate>
-              <Title numberOfLines={1}>
-                {artwork.title}
-              </Title>
-              {!!artwork.date && <Date>{`, ${artwork.date}`}</Date>}
+              {/* Nested Text components are necessary for the correct behaviour on both short and long titles + dates */}
+              <Text numberOfLines={1} ellipsizeMode={"middle"} style={{ fontFamily: fonts["garamond-italic"] }}>
```

Comment:
These could be styled and it would look pretty reasonable to read- 

```
const Subtitle = styled.Text`
   font-family: ${fonts["garamond-italic"]};
   font-size: 14;
 `
const Italics = styled.Text`
   font-family: ${fonts["garamond-regular"]};
 `

...

<Subtitle numberOfLines={1} ellipsizeMode={"middle"}>Thing<Italics>{`, ${artwork.date}`}</Italics></Subtitle>
```